src/SFA.DAS.ProviderRelationships.Database/Scripts/PostDeployment/SeedDevData.sql

```diff
@@ -0,0 +1,4 @@
+﻿IF (NOT EXISTS (SELECT 1 FROM [dbo].[Account] WHERE Id = 1))
```

Comment:
Should be [dbo].[Accounts] - this stops the db publishing!
```suggestion
IF (NOT EXISTS (SELECT 1 FROM [dbo].[Accounts] WHERE Id = 1))
```